components/ntp_background_images/browser/view_counter_service.cc

```diff
@@ -468,6 +476,53 @@ void ViewCounterService::MaybeTriggerNewTabPageAdEvent(
                                          /*intentional*/ base::DoNothing());
 }
 
+void ViewCounterService::MaybeTriggerSponsoredRichMediaAdEvent(
+    const std::string& placement_id,
+    const std::string& creative_instance_id,
+    const std::string& ad_event_type) {
+  if (!ads_service_) {
+    // If `brave_ads::kShouldAlwaysRunBraveAdsServiceFeature` flag is disabled,
```

Comment:
this seems like a detail that `ViewCounterService` doesn't need to know about and this comment will probably get outdated and not be updated

Reply:
Removed comments.